.travis.yml

```diff
@@ -1,25 +0,0 @@
-language: node_js
-node_js: node
-sudo: required
-script:
-- npm run lint
-- npm run test:unit
-- 'if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then npm run test:sauce; fi'
-after_success:
-- npm run report-coverage
-- bash ./update.sh
-addons:
-  chrome: stable
-cache:
-  yarn: true
-  directories:
-  - node_modules
-env:
-  global:
-  - SAUCE_USERNAME: Desire2Learn
-  # GITHUB_RELEASE_TOKEN
-  - secure: oOypFGViw2h3sU3U8MWAFNYb9E6KY+EX4Ltcd6AK1/NVT+7m+SKuTR6HfWcYMYSGZtJUA9I68LepGhbMQXmOEVOxf1d2kcmmcvqldzBon+MHeZ8ozvOMiQj/UYY2GpglSXjin/6kW+f4g92TX4Rio6x2fnj1fYj48k4HqBRPET/6j32Itlw+0q0VJ+D7CUugUtzMC7JDhdrfugyvUs1iy3GQnkeMIPoswLhcyDQ+VH/BAZvtzjCbV7qvUvs3JdPZg948SdGWfGJBrmc48u16PstldqWrKsDFtlvkFvWeFsM5loQybXH546fviED2WFRYLUT+bXItUxvyU5AkxZMMnU1OMO+Qu2UWPq+jyPdckDlZeidcNrk2ywYYCwnz5l8ngWK4xMDlTS5soyTnd4VZi36OxAkryzRX9+l+hgYOVt/fsXbjBVP4zTcPi2+1j2Z9BeqJ1eUHFOUSiXDckhokyIdzvB7Ar5Ze+4MA4RwjP4HR3qWWQmRVI/husQcuAnYNRxqF/Tw1r08KyRdj2BF3lmD6hFS1hR0vN+S8J0UeLLINZGsHGVC96qscGzacc0DCnntHtcOb/EN5qN6aQNz5DJg+kpslTZGm8QYKeIz2J/a3UwoZcM2FgBhQCvehQ5BA4kiXeNbWHMGlaeC26OteDgV/g4wtt6x60FnPivrNVf4=
-  # SAUCE_ACCESS_KEY
-  - secure: cpu4DdXddutV8VXGwkCiR99jiUwsGiUMshbMhceNJJ7LAjrZCSe9y8uMVIAIwGOjm3Qh8I4arjQjPSdQB9AqsnfpNso1mqMOfoGe4bc9TktnlC24RTE19qemtU42yqt9ktxnPojH0+4/V7u0b4fmbbKzXVMpXt7pt+oB1t7fY7r5cSci7gjTeiajccRRceOs2GD9p/MHiGaW/IP7yqj0QqLFyhGFaP+3mY2VFIL2rBGo/kOvSuRwkM5ACJVSxy1zGTombWRjVBGTqWM6vPiJbCHoBR7hSGwm09V4olic7tzJam5ozKy/UfhDGMihkZsRPsdF/TjZHjxdRr3Uwd/GU6L9BDYbG5fw7YMoMQyKxjEJT1FnO5vTBrQmvZvR4WTG8TJ3gyJTmxvuu2S3zsBQVhdMHfXcmnglrc2hOFWAay/yF/ReKVa5+KiD58pRmfbvI/Szd6UIDKECX/8JlbNxFH+zTyvTuJ3oLcXQZUXNGMTLZPofFvOPLZG5gu1Jt2Op9RG3oZ4RlTZXXuBbAdtlihFJdHo6EMHxbDThiKMb7ZJZSH0bBsW2KgorPM/ES4pNOFv/5U5ZhyEza+rDUWXq68s6rnqVsQfeAb4/dil7ZNpXF/JjM45TkgaBdAuSUKtvJNdmY9OiZviKrXhSHKYEsgXDPeosIeShtH0gkHNDtzk=
-  # CDN_SECRET
```

Comment:
So was this CDN_SECRET not used for anything?

Reply:
From what I can tell this used to publish to the CDN a long time ago, but that was all ripped out during the Polymer 3 upgrade: https://github.com/Brightspace/d2l-telemetry-browser-client/pull/17/files